### PR TITLE
Add timeline for docs site

### LINKED ISSUE #140: Add timeline example

### PR DESCRIPTION
**Pull request by beerinho**
Closes #140 

~I couldn't figure out the best way to do the imports so please let me know if there is a better way.~ fixed by moving to a monorepo

You can see it locally by running `pnpm docs:dev`
Screenshot 2024-07-26 at 17 08 58
Screenshot 2024-07-26 at 16 31 34

### CODE REVIEW

**Comment by beerinho** [comment]
I'm able to build this locally just fine so I'm not sure what the problem is on netlify

File: apps/docs/src/content/docs/Timeline.svelte [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			idle={(element.start / total) * wrapper_width}
+		/>
+	{/each}
+</div>
+
+<style>
+	.wrapper {
+		height: 400px;
+		position: relative;
+		background: white;
```
Comment (paoloricciuti) [inline apps/docs/src/content/docs/Timeline.svelte:80]:
```suggestion
		background: transparent;
```

File: apps/docs/src/content/docs/Timeline.svelte [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		overflow: hidden;
+	}
+	.line {
+		position: absolute;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		transform: translateX(calc(var(--left) * 1px));
+		width: 100px;
+		border-left: 2px solid black;
```
Comment (paoloricciuti) [inline apps/docs/src/content/docs/Timeline.svelte:90]:
```suggestion
		border-left: 2px solid var(--sl-color-text);
```

File: apps/docs/src/content/docs/Timeline.svelte [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+		background-color: white;
```
Comment (paoloricciuti) [inline apps/docs/src/content/docs/Timeline.svelte:91]:
```suggestion
		background-color: transparent;
```

File: apps/docs/src/content/docs/TimelineItem.svelte [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+		border-radius: 25px 0 0 25px;
+		display: flex;
+		align-items: center;
+		justify-content: start;
+		margin: 0;
+		cursor: pointer;
+		overflow: hidden;
+		background: transparent;
+		color: var(--sl-color-black);
+		border-left: 2px solid var(--sl-color-black);
```
Comment (paoloricciuti) [inline apps/docs/src/content/docs/TimelineItem.svelte:70]:
```suggestion
		color: var(--sl-color-text);
		border-left: 2px solid var(--sl-color-text);
```

File: apps/docs/src/content/docs/TimelineItem.svelte [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+		color: var(--color-canceled);
+	}
+
+	.dropped .status {
+		margin-left: 5px;
+	}
+
+	.queued {
+		border: 5px var(--color-waiting) solid;
+		background-color: var(--sl-color-black);
```
Comment (paoloricciuti) [inline apps/docs/src/content/docs/TimelineItem.svelte:95]:
```suggestion
		background-color: var(--sl-color-text);
```

**Review by paoloricciuti — COMMENTED** [review]
Just some css var comments...we can check them together if you want (i don't know if i actually suggested anything or more than needed 😄  )

**Review by paoloricciuti — APPROVED** [review]

**Pull request merged**